portfolio/src/main/webapp/script.js

```diff
@@ -57,39 +63,83 @@ function addProjectDescription(project) {
   wrapper.style.backgroundColor = projectNameToData[projectName]['backgroundColor']; 
 }
 
-
 /**
  * This is the helper function for addProjectDescription.
- * This function creates a data key-value pair object for each project.
- * Input: 
+ * This function creates a data key-value pair object for each project. 
  */
 function createProjectData(projectDetail, backgroundColor) {
- 
   projectData = {}
   projectData['projectDetail'] = projectDetail;
   projectData['backgroundColor'] = backgroundColor; 
-
   return projectData; 
 }
 
 /**
  * This function enables the highlighting of a selected section from the navigation bar. 
  * This function is called by the clicking of any one of the boxes in the navigation bar.
  * Input: section to select
- * Effect: selected section will have their border shown, and all the other sections will not have the border. 
+ * Effect: selected section will have their border shown, and all the other sections 
+ *         will not have the border. 
  */
 function highlightSection(selectedIdString) {
-
   document.getElementById(selectedIdString).style.border = 'solid';
   document.getElementById(selectedIdString).style.borderColor = '#c5cf0a';
 
   const navbarSections = document.getElementsByClassName('navbarSection');
 
-  Object.keys(navbarSections).forEach(
-    (key) => {
-      const section = navbarSections[key];
+  Object.values(navbarSections).forEach(
+    (section) => {
       if (section.id !== selectedIdString) {
         section.style.border = 'none'; 
       }
     }); 
+}
+
+/**
+ * This function fetches the text from /data.
+ * Then, the function puts the received text into the quoteWrapper. 
+ */
+function getQuotes() {
+  fetch('/data')
+      .then(response => response.json())
+      .then((quotesJson) => {
+        const quoteListElement = document.getElementById('quoteWrapper');
+        quoteListElement.innerHTML = '';
+
+        Object.values(quotesJson).forEach(
+          (quote) => {
+            quoteListElement.appendChild(createQuotesListElement(quote)); 
+          });        
+      });
+}
+
+/**
+ * This is a helper function, called inside getQuotes. 
+ * This function creates a li element that contains each individual quote;
+ * this li element will be added to the Quotes section in the HTML file.  
+ */
+function createQuotesListElement(quote) {
+  const liElement = document.createElement('li');
+  liElement.innerText = quote;
+  return liElement;
+}
+
+/**
+ * This function performs a POST request to deleteQuote servlet, which 
+ * handles the deletion of all quotes in Datastore. 
+ */
+function cleanData() {
+  fetch('/deleteQuote', {method: 'POST'})
+    .then(() => {
+      refreshQuotes(); 
+  });
```

Comment:
I think we need to indent this line by two spaces.

Reply:
Done.